src/lib/journey/stages/_utilities/initializeProtect.ts

```diff
@@ -0,0 +1,30 @@
+import { journeyStore } from '$journey/journey.store';
+import { PIProtect } from '@forgerock/ping-protect';
+import { get } from 'svelte/store';
+
+async function initializeProtect() {
+  const store = get(journeyStore);
+  /**
+   * At the very least we need an env id. So lets check if we have one.
+   * If we do, we can initialize PingProtect. If we do, we can initialize PingProtect.
+   * We are going to use these defaults
+   * and overwrite them with whatever we have from the
+   * start config call. If we have initialized set to false we can proceed
+   */
+  if (store?.pingProtect?.envId) {
+    try {
+      await PIProtect.start({
+        envId: store.pingProtect?.envId,
+        consoleLogEnabled: store.pingProtect?.consoleLogEnabled,
+      });
+    } catch (error) {
+      console.error(error);
```

Comment:
Can we improve the error messaging here? The error that is returned by the Signals SDK may not be obvious to the dev, so we should probably wrap this error message with a bit more context.

Reply:
Do we want an error? I wasn't fully sure what we wanted in the fail case here. Since we aren't trying to stop the application, is a console.error worthwhile?

Is it better to just ignore it?

Reply:
I think we should log the error, yes. I don't want a silent error to happen as we'd want to bring the dev's attention to the issue. Let's just prefix the logged error with context helping point the dev in the right direction.

Reply:
no longer applicable